hmpps-prisoner-search/src/main/resources/application-client-fields.yml

```diff
@@ -0,0 +1,11 @@
+search.clients:
+  restricted-patients-client:
```

Comment:
list of fields obtained from restricted patients dto - https://github.com/ministryofjustice/hmpps-restricted-patients-api/blob/54cc87507b653d6ff6b81c0e778bf90f20f75712/src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsrestrictedpatientsapi/model/response/PrisonerResult.kt#L6-L16